src/pages/components/modal/style.mdx

```diff
@@ -193,3 +210,31 @@ the content while sticking to the bottom of the mobile screen.
 
 </Column>
 </Row>
+
+## AI presence
+
+The following are the unique styles applied to the component when the AI slug is
+present. Unless specified, all other tokens in the component remain the same as
+the non-AI variant.
+
+More information about AI style elements is coming soon.
+
+| Element                    | Property         | Token / Size        |
+| -------------------------- | ---------------- | ------------------- |
+| Modal:background           | background-color | `layer-01`          |
```

Comment:
This would be a contextual layer token, like a field.  
```suggestion
| Modal:background           | background-color | `$layer`*          |
```

Reply:
```suggestion
| Modal:background           | background-color | `$layer` \*         |
```